ext/cl_khr_command_buffer.asciidoc

```diff
@@ -490,6 +490,19 @@ with the command-queue used on command-buffer creation. Where a _compatible_
 command-queue is defined as a command-queue with identical properties targeting
 the same device and in the same OpenCL context.
 
+While constructing a command-buffer it is valid for the user to interleave calls
+to the same queue which create commands, such as {clCommandNDRangeKernelKHR}, with
+queue submission calls, such as {clEnqueueNDRangeKernel} or
+{clEnqueueCommandBufferKHR}. That is, there is no effect on queue state from
+recording commands. The purpose of the queue parameter is to define the device
+and properties of the command, which are constant queries on the queue object.
+
+A command-buffer object should increment the reference count of OpenCL objects
+such as queues, buffers, images, and kernels referenced in commands recorded to
+the command-buffer. This enables correct behaviour of the command-buffer when the
```

Comment:
Minor: we've been using the US English spelling elsewhere in the spec.

```suggestion
the command-buffer. This enables correct behavior of the command-buffer when the
```

Reply:
Done, there were also a couple places in the mutable-dispatch spec where "behaviour" had crept in that I've updated